eng/tools/rush-runner.js

```diff
@@ -1,51 +1,131 @@
-const fs = require('fs');
-const path = require('path');
-const process = require('process');
-const { spawnSync } = require('child_process');
+const fs = require("fs");
```

Comment:
Nit: Can you please undo the change from `'` to `"` (and any other formatting changes)?  This was probably done automatically by your editor.

Reply:
We could add `/eng/**/*` to our Prettier ignore file to prevent editors from doing this to files that are owned by EngSys. Our root `/.prettierrc.json` is configured to enforce double quotes.

Reply:
I think it makes the most sense to use the same formatting for all JS files used by our team, including stuff under `/eng`.  @praveenkuttappan: Can you update all the JS files under `/eng` to the new formatting in a separate PR?

Reply:
Sure. Will do it. @mikeharder : Does that mean we don't need to revert formatting changes in this PR?

Reply:
I prefer to put formatting changes in separate PRs, but I don't feel that strongly.  Up to you.